CorsixTH/Lua/app.lua

```diff
@@ -945,13 +945,8 @@ function App:saveConfig()
           -- replace the line, if needed
           handled_ids[identifier] = true
           if value ~= tostring(self.config[identifier]) then
-            local new_value = self.config[identifier]
-            if type(new_value) == "string" then
-              new_value = string.format("[[%s]]", new_value)
-            else
-              new_value = tostring(new_value)
-            end
-            lines[#lines] = string.format("%s = %s", identifier, new_value)
+            local new_value, options = self.config[identifier], { start_at_one = true }
```

Comment:
Why did you keep `new_value`  as variable name? It never contains a new value after your changes.